src/StreamingDataFrames/streamingdataframes/dataframe/dataframe.py

```diff
@@ -71,19 +72,22 @@ class StreamingDataFrame:
     """
 
     def __init__(
-        self, topics: List[Topic], _pipeline: Pipeline = None, _id: str = None
+        self,
+        topics_in: List[Topic],
+        topics_out: Optional[List[Topic]] = None,
```

Comment:
The meaning of this parameter is a bit unclear to me.
Do we now expect users to provide the list of output topics on `StreamingDataFrame.__init__`?

Reply:
Removed as an argument; now handled automatically by the runner